stories/tables/sortable-table.story.js

```diff
@@ -87,10 +87,17 @@ storiesOf('SortableTable', module)
   .add('with custom cell component', () => (
     <SortableTable data={tableData}>
       <Column name="name" />
-      <Column name="age" />
+      <Column name="age"  />
       <Column name="active" component={CustomCell} />
     </SortableTable>
   ))
+  .add('with valid DOM properties on cell component per column', () => (
+    <SortableTable data={tableData}>
+      <Column name="name" tabIndex='tab-index'/>
```

Comment:
Can you please pass a valid value for tab-index? https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex